tests/compat.sh

```diff
@@ -799,16 +798,13 @@ wait_client_done() {
     echo "EXIT: $EXIT" >> $CLI_OUT
 }
 
-# run_client <name> <cipher>
+# run_client PROGRAM_NAME STANDARD_CIPHER_SUITE PROGRAM_CIPHER_SUITE
 run_client() {
     # announce what we're going to do
     TESTS=$(( $TESTS + 1 ))
-    VERIF=$(echo $VERIFY | tr '[:upper:]' '[:lower:]')
-    TITLE="`echo $1 | head -c1`->`echo $SERVER_NAME | head -c1`"
+    TITLE="${1%"${1#?}"}->${SERVER_NAME%"${SERVER_NAME#?}"}"
     TITLE="$TITLE $MODE,$VERIF $2"
-    printf "%s " "$TITLE"
-    LEN=$(( 72 - `echo "$TITLE" | wc -c` ))
-    for i in `seq 1 $LEN`; do printf '.'; done; printf ' '
+    printf "%s %.*s " "$TITLE" "$((72 - ${#TITLE}))" ........................................................................
```

Comment:
This adds one more dot than before. My mistake (I didn't notice that `echo "$TITLE" | wc -c` is one plus the length of `$TITLE`, due to counting the trailing newline). Not a bug, but makes the line gratuitously longer and makes the comparison with old logs harder.
```suggestion
    printf "%s %.*s " "$TITLE" "$((71 - ${#TITLE}))" ........................................................................
```

Reply:
Thanks for letting me know. I also noticed this adds one more dot in the output. I thought it's only a way to display the results so I didn't mention it. As you said now `compat.sh` is no longer aligned with `ss-opt.sh`, I also prefer to keeping the same number of dots as before although you have approved this PR.